2022-03-27 22:00:45,121 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2022-03-27 22:00:45,122 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from /home/achyuth/.local/lib/python3.9/site-packages/instabot/bot/bot.py
2022-03-27 22:00:45,124 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2022-03-27 22:00:46,198 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [200]>
2022-03-27 22:00:47,208 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [200]>
2022-03-27 22:00:48,226 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: launcher/sync/ returned response: <Response [200]>
2022-03-27 22:00:50,256 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/contact_point_prefill/ returned response: <Response [200]>
2022-03-27 22:00:53,159 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/login/ returned response: <Response [400]>
2022-03-27 22:00:53,160 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"challenge_required","challenge":{"url":"https://i.instagram.com/challenge/AXEM49HeVwK9RB8t_NlzGq7n3vkGgNoj7vIRhHuM5soBbAleH9wNpuEVsdefnBKsk0gHyhI/ucOMZHdqQpXvzNi3/","api_path":"/challenge/AXEM49HeVwK9RB8t_NlzGq7n3vkGgNoj7vIRhHuM5soBbAleH9wNpuEVsdefnBKsk0gHyhI/ucOMZHdqQpXvzNi3/","hide_webview_header":true,"lock":true,"logout":false,"native_flow":true,"flow_render_type":0},"status":"fail","error_type":"checkpoint_challenge_required"}'
2022-03-27 22:00:53,160 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2022-03-27 22:00:53,161 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: challenge_required
2022-03-27 22:00:53,161 - instabot version: 0.117.0 (api) - INFO - Error type: checkpoint_challenge_required
2022-03-27 22:00:58,241 - instabot version: 0.117.0 (api_photo) - ERROR - Photo Upload failed with the following response: <Response [403]>
2022-03-27 22:00:58,246 - instabot version: 0.117.0 (bot_photo) - INFO - Photo '/home/achyuth/Downloads/demo.jpg' is not uploaded.
